K59/Nguyen-Van-Sang/computer_network/VLAN, VXLAN.md

```diff
@@ -0,0 +1,281 @@
+
+# Chương 1: VLAN 
+
+## 1.1 Lịch sử ra đời
+Với mạng LAN, do sự giới hạn về đường truyền vật lí ta chỉ có thể kết nối các máy tính trong cùng một địa điểm (cùng phòng...) lại với nhau để tạo thành mạng 1 mạng LAN với thiết bị trung tâm là hub hoặc switch. 
+
+Tuy nhiên trong thực tế số lượng máy tính trong một mạng LAN thường không nhiều, ngoài ra các máy tính trong cùng một phòng có thể thuộc các mạng LAN khác nhau vì vậy tốn nhiều tài nguyên hub, switch trong khi lại chưa tận dụng hết số cổng trên các thiết bị này. Hơn nữa các máy tính thuộc các đơn vị khác nhau (ở các địa điểm khác nhau) của cùng một phòng ban sẽ có các nhu cầu trao đổi dữ liệu chủ yếu với nhau.
+
+Để giải quyết những vấn đề này, giải pháp đưa ra là nhóm các máy tính thuộc các LAN khác nhau vào cùng một switch, các máy thuộc các switch khác nhau có thể nằm chung một LAN. Giải pháp này gọi là LAN ảo hay VLAN.
+
+## 1.2 Khái niệm VLAN và lợi ích VLAN đem lại
+
+**VLAN**
+
+VLAN là tên viết tắt của Virtual Local Area Network hay còn được gọi là mạng LAN ảo. VLAN là một kỹ thuật cho phép tạo một mạng LAN độc lập một cách logic trên cùng một kiến trúc vật lí. Việc tạo ra các VLAN giúp giảm thiểu số broadcast domain (miền quảng bá) và tạo điều kiện thuận lợi cho quản lý một mạng cục bộ rộng lớn.
+
+**Broadcast domain**
+> Broadcast domain: Là một không gian logic mà các máy tính trong đó có thể trao đổi thông tin với nhau bằng các frame (mức data link layer). Trong một broadcast domain, khi một máy tính gửi một frame thì frame đó sẽ được gửi tới tất cả các máy khác trong cùng miền này. 
++ Khi chưa có VLAN: Thiết bị giới hạn miền quảng bá thường là các Router (do cơ chế chặn các bản tin quảng bá). Và cũng chính Router tạo ra các miền quảng bá bằng việc mỗi một giao diện của Router là một Broadcast domain.
+
++ Khi áp dụng VLAN: Mỗi một VLAN là một miền broadcast. Một switch không hỗ trợ VLAN thì toàn bộ các máy tính, thiết bị cắm vào switch đó nằm trong một miền broadcast.
+
+
+
+Ví dụ: 
+Hình 1.1
+
+![h1](imageNet/h1net.png_45kl0vdr1p)
+
+Xét mô hình 3 tầng trong một tòa nhà như Hình 1.1, ta thấy tại mỗi tầng đều có các máy tính (hoặc thiết bị) thuộc cả 3 phòng ban(Engineering, Marketing, Accounting). Như vậy nếu không sử dụng kĩ thuật VLAN, mà muốn cho các thiết bị trong cùng 1 phòng ban có thể trao đổi dữ liệu nội bộ với nhau ta cần 9 switch (3 switch cho mỗi tầng) và số miền quảng bá tương ứng sẽ là 9 miền. Mặt khác như hình trên, khi áp dụng VLAN ta thu được chỉ còn 3 miền broadcast mà các máy thuộc 3 tầng ở cùng một phòng ban vẫn có thể trao đổi dữ liệu với nhau.
+
```

Comment:
"Như vậy nếu không sử dụng kĩ thuật VLAN, mà muốn cho các thiết bị trong cùng 1 phòng ban có thể trao đổi dữ liệu nội bộ với nhau ta cần 9 switch (3 switch cho mỗi tầng) và số miền quảng bá tương ứng sẽ là 9 miền"

Có thật không ? Phải dùng tới 9 con switch cơ à ? Em vẽ ra cho anh xem xem là dùng 9 con đấy kiểu gì để chúng nó trao đổi dữ liệu nội bộ với nhau xem nào ?

"Mặt khác như hình trên, khi áp dụng VLAN ta thu được chỉ còn 3 miền broadcast mà các máy thuộc 3 tầng ở cùng một phòng ban vẫn có thể trao đổi dữ liệu với nhau."

Em giải thích hình vẽ kia cho anh được không ? . Vẽ một hình chi tiết đánh số các cổng, gán IP cho từng máy trong toàn bộ hệ thống (em tự chọn), giải thích **chi tiết** cho anh quá trình một gói tin nó đi từ một  máy ở Engineering VLAN sang Marketing VLAN như thế nào ?